spec.json

```diff
@@ -0,0 +1,690 @@
+          "Shorthand": "c",
+          "Usage": "path/to/config.yml where parameters can be set instead of on the command line"
+        },
+        "connections": {
+          "Name": "connections",
+          "Usage": "path/to/connections.yml that contains connections that are used in the reload. Note that when specifying connections in the config file they are specified inline, not as a file reference!"
+        },
+        "dimensions": {
+          "Name": "dimensions",
+          "Usage": "A list of generic dimension json paths"
+        },
+        "engine": {
+          "Name": "engine",
+          "Shorthand": "e",
+          "Usage": "URL to engine (default \"localhost:9076\")"
+        },
+        "headers": {
+          "Name": "headers",
+          "Usage": "Headers to use when connecting to qix engine",
+          "DefValue": "[]"
+        },
+        "measures": {
+          "Name": "measures",
+          "Usage": "A list of generic measures json paths"
+        },
+        "objects": {
+          "Name": "objects",
+          "Usage": "A list of generic object json paths"
+        },
+        "script": {
+          "Name": "script",
+          "Usage": "path/to/reload-script.qvs that contains a qlik reload script. If omitted the last specified reload script for the current app is reloaded"
+        },
+        "silent": {
+          "Name": "silent",
+          "Usage": "Do not log reload progress",
+          "DefValue": "false"
+        },
+        "ttl": {
+          "Name": "ttl",
+          "Usage": "Engine session time to live in seconds",
+          "DefValue": "30"
+        }
+      }
+    },
+    "catwalk": {
+      "Use": "catwalk",
+      "Short": "Opens the specified app in catwalk",
+      "Long": "Opens the specified app in catwalk. If no app is specified the catwalk hub will be opened.",
+      "Flags": {
+        "app": {
+          "Name": "app",
+          "Shorthand": "a",
+          "Usage": "App name, if no app is specified a session app is used instead."
+        },
+        "catwalk-url": {
+          "Name": "catwalk-url",
+          "Usage": "Url to an instance of catwalk, if not provided the qlik one will be used.",
+          "DefValue": "https://catwalk.core.qlik.com"
+        },
+        "config": {
+          "Name": "config",
+          "Shorthand": "c",
+          "Usage": "path/to/config.yml where parameters can be set instead of on the command line"
+        },
+        "engine": {
+          "Name": "engine",
+          "Shorthand": "e",
+          "Usage": "URL to engine (default \"localhost:9076\")"
+        }
+      }
+    },
+    "eval": {
+      "Use": "eval",
+      "Short": "Evaluates a list of measures and dimensions",
+      "Long": "Evaluates a list of measures and dimensions. To evaluate a measure for a specific dimension use the \u003cmeasure\u003e by \u003cdimension\u003e notation. If dimensions are omitted then the eval will be evaluated over all dimensions.",
```

Comment:
This looks weird, I think we need to redesign the `eval` command so it uses flags for dimensions/measures instead of string parsing